pkg/chain/ethereum/tbtc.go

```diff
@@ -1,17 +1,84 @@
 package ethereum
 
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/keep-network/keep-common/pkg/chain/ethereum"
+	"github.com/keep-network/keep-core/pkg/chain"
+	"github.com/keep-network/keep-core/pkg/chain/ethereum/ecdsa/gen/contract"
+	"github.com/keep-network/keep-core/pkg/operator"
+)
+
+// Definitions of contract names.
+const (
+	WalletRegistryContractName = "WalletRegistry"
+)
+
 // TbtcChain represents a TBTC-specific chain handle.
 type TbtcChain struct {
 	*Chain
+
+	walletRegistry *contract.WalletRegistry
 }
 
 // NewTbtcChain construct a new instance of the TBTC-specific Ethereum
 // chain handle.
 func newTbtcChain(
+	config ethereum.Config,
 	baseChain *Chain,
 ) (*TbtcChain, error) {
-
+	// TODO: once WalletRegistry contract can be deployed locally, use it.
+	// For now set walletRegistry to nil.
```

Comment:
We have the bindings generated so I think we should be able to implement this part. If the beacon is checked as the first one, and WalletRegistry is not available, the clients should be able to connect anyway.

Reply:
Done in 33800725ed4285fd93543f0bcc1c80cc7b8238ba.
This works, but on the client start you need to have an entry for wallet registry address in the toml config files with some dummy value:
```
  [ethereum.ContractAddresses]
  (...)
  WalletRegistry = "0x0000000000000000000000000000000000000000"
  (...)
 ```
 
 Otherwise there is an error:
 ```
 2022-07-14T14:48:47.745+0200    FATAL   keep-main       keep-core/main.go:76  error connecting to Ethereum node: [could not create TBTC chain handle: [failed to resolve WalletRegistry contract address: [no address information for [WalletRegistry] in configuration]]] 
 ```